atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionTest.java

```diff
@@ -1309,6 +1318,149 @@ public void testGetRangesPaging() {
         verifyAllGetRangesImplsNumRanges(t, ranges, ImmutableList.of("v"));
     }
 
+    @Test
+    public void getRowsAccessibleThroughCopies() {
+        Transaction t = startTransaction();
+        byte[] rowKey = row(0);
+        byte[] value = value(0);
+        t.put(TEST_TABLE, ImmutableMap.of(
+                Cell.create(rowKey, column(0)), value));
+        t.commit();
+
+        t = startTransaction();
+        SortedMap<byte[], RowResult<byte[]>> result =
+                t.getRows(TEST_TABLE, ImmutableList.of(rowKey), ColumnSelection.all());
+        assertThat(result.get(rowKey))
+                .as("it should be possible to get a row from getRows with a passed-in byte array")
+                .isNotNull()
+                .satisfies(rowResult ->
+                        assertThat(rowResult.getOnlyColumnValue()).isEqualTo(value));
+
+        byte[] rowKeyCopy = row(0);
```

Comment:
nit: i think this is more clear as rowKey.clone()

Reply:
i suppose, same as in other places

Reply:
Alright, yeah agree that reads better. Will change.